prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullname = 'Heather Merizan';
+    console.log(fullname);
+    var numberofpets = '2';
+    console.log(numberofpets);
+    var likesVegetables = 'True';
+    console.log(likesVegetables);
+    var nothing = 'null';
```

Comment:
Hi Heather! For each of `2`, `true`, and `null`, you want to leave out the quotes so that the JavaScript engine doesn't think you mean string values.